go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/opencontainers/runtime-tools v0.9.0
 	github.com/opencontainers/selinux v1.4.0
 	github.com/openshift/api v0.0.0-20200106203948-7ab22a2c8316
-	github.com/openshift/imagebuilder v1.1.3
+	github.com/openshift/imagebuilder v1.1.4-0.20200325224900-8de717452756
```

Comment:
I'd prefer having a full imagebuilder release here if we can.  I know @carbonin can't make that happen from that end.

Reply:
Your wish is my command.
https://github.com/openshift/imagebuilder/releases/tag/v1.1.4